webservices/resources/elections.py

```diff
@@ -176,6 +176,7 @@ def _get_pairs(self, totals_model, kwargs):
     def _get_latest(self, pairs):
         latest = db.session.query(
             pairs.c.cash_on_hand_end_period,
+            pairs.c.candidate_id,
```

Comment:
I saw your note on this too, thank you for indicating why this is now needed!  Just to verify, this change doesn't actually change the query construction or performance?

Reply:
Yes, you're right!  We were relying on the `distinct` clause to add the column into the `SELECT` portion of the query automatically, as noted in the piece of the migration guide you linked to.  Thanks!